cpp/include/cudf/io/parquet_metadata.hpp

```diff
@@ -180,8 +180,10 @@ class parquet_metadata {
  public:
   /// Key-value metadata in the file footer.
   using key_value_metadata = std::unordered_map<std::string, std::string>;
-  /// row group metadata from each RowGroup element.
+  /// Row group metadata from each RowGroup element.
   using row_group_metadata = std::unordered_map<std::string, int64_t>;
+  /// Column chunk metadata from each ColumnChunkMetaData element.
```

Comment:
You started it, with the capitalization :D
```suggestion
  /// Row group metadata from each RowGroup element
  using row_group_metadata = std::unordered_map<std::string, int64_t>;
  /// Column chunk metadata from each ColumnChunkMetaData element
```